test-app/.travis.yml

```diff
@@ -0,0 +1,28 @@
+---
+language: node_js
+node_js:
+  - "10"
+
+dist: xenial
+
+addons:
+  chrome: stable
+
+cache:
+  yarn: true
+
+env:
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+
+branches:
+  only:
+    - master
+
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
How come this is using `yarn` while the rest of the project is pnpm?

Reply:
.travis is from some default boilerplate from the migrator tool that I forgot to remove

Reply:
removed